### DISCUSSION TITLE
Super Game Boy BIOS Dump?

### DISCUSSION THREAD

**Discussion by 7ank0v1c** [sanni/cartreader#676] [Need help]
Heya, I have just order a super game boy cart for the SNES. But I was wondering if the Sanni can dump the bios from it? 

Or is this something which could be implemented later on if not?

**Comment by Cephiros** [comment]
Yes, at least the US released variant. I haven't tested any other regions, or variants like SGB2

**Reply by 7ank0v1c** [reply]
Heya, so mine arrived today (Japanese). I dumped it, but im not able to use it for SGB emulation. Are you sure it is the BIOS/boot rom that it is dumping?

**Reply by 7ank0v1c** [reply]
I think it might be the emulator I am using at this point to be honest. I will confirm more later

**Reply by PsyK0p4T** [reply]
"Adapters" SNES carts can be dumped fine as far as I know.
I tried with the Super Game Boy, and other Satella receivers like BS-X or "Same Game" carts.

**Reply by Cephiros** [reply]
I only had problems dumping game genie, I think even the xband dumped and identified.

**Reply by 7ank0v1c** [reply]
I can confirm I got it working, it was indeed the emulator 😊 

Love that we are able to do this using the Sanni
